app/services/notices/setup/prepare-return-forms.service.js

```diff
@@ -13,20 +13,29 @@ const SessionModel = require('../../../models/session.model.js')
 /**
  * Orchestrates fetching and presenting the data for the return form
  *
- * @param {string} sessionId
+ * @param {string} sessionId - The UUID of the current session
+ * @param {string} returnId - The UUID of the return id
```

Comment:
Being pedantic, but it isn't a UUID (unfortunately!)

Also, the legacy code often trips itself up by not being specific; for example, 'return' might refer to the return version, the return requirement, or the return log.

It's only a few more chars in this case, and one to bear in mind when referring to 'returns'.

```suggestion
 * @param {string} returnLogId - The return log ID
```